src/main/java/seedu/address/logic/commands/AddToConsultCommand.java

```diff
@@ -0,0 +1,179 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_CONSULTATIONS;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.consultation.Consultation;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Adds students to an existing consultation in the consultation list.
+ */
+public class AddToConsultCommand extends Command {
+    public static final String COMMAND_WORD = "addtoconsult";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a student to a consultation at INDEX."
+        + "Parameters: INDEX (must be a positive integer)"
+        + "[" + PREFIX_NAME + "NAME]...\n"
+        + "Example: " + COMMAND_WORD + " "
+        + PREFIX_NAME + "John Doe ";
+    public static final String MESSAGE_SUCCESS = "New student(s) added to consult at index %1$s: %1$s";
+    public static final String MESSAGE_DUPLICATE_STUDENT = "The student(s) added are already in the consultation";
+    public static final String MESSAGE_PERSON_NOT_FOUND = "No student match given name(s)";
```

Comment:
No student matching the given name(s)? I guess its just small grammar issues

Reply:
resolved!